pages.ta/common/calc.md

```diff
@@ -0,0 +1,36 @@
+# calc
+
+> முனையத்தில் ஒரு ஊடாடும் தன்னிச்சையான துல்லியமான கால்குலேட்டர்.
+> மேலும் தகவல்: <https://github.com/lcn2/calc>.
+
+- ஊடாடும் பயன்முறையில் `calc` ஐத் தொடங்கவும்:
+
+`calc`
+
+- ஊடாடாத பயன்முறையில் கணக்கீடு செய்யவும்:
+
+`calc '{{85 * (36 / 4)}}'`
+
+- வெளியீட்டு வடிவமைப்பு இல்லாமல் கணக்கீடு செய்யுங்கள் (குழாய்களுடன் பயன்படுத்த):
+
+`calc -p '{{4/3 * pi() * 5^3}}'`
+
+- ஒரு கணக்கீட்டைச் செய்து, பின்னர் [i]ஊடாடும் பயன்முறைக்கு மாறவும்:
+
+`calc -i '{{sqrt(2)}}'`
+
+- ஒரு குறிப்பிட்ட அனுமதி [m]ode இல் `calc` ஐத் தொடங்கவும் (0 முதல் 7, இயல்புநிலை 7 வரை):
+
+`calc -m {{mode}}`
```

Comment:
Maybe this could also be translated, but I'm not sure if this makes sense.

Reply:
I too have the same confusion because there are multiple words used in place of mode but it changes for different use cases. Sometimes those words might mean different, mode generally means முறை which means method. But I am not sure whether this one can be translated.

Reply:
I think `முறை` is fine.